charts/dave/charts/document-storage/templates/configmap.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "daveDocumentStorage.fullname" . }}
+  labels: 
+    {{- include "daveDocumentStorage.labels" . | nindent 4 }}
+data:
+  {{- with .Values.extraEnvVars.env }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)